lib/ethereum/wss_client.rb

```diff
@@ -0,0 +1,149 @@
+require 'open3'
+require 'json'
+
+module Ethereum
+  class WssConnection
+    attr_accessor :host,:ws_in,:ws_out,:ws_err,:wait_thr,:index
```

Comment:
also should have an ssl attribute.

```suggestion
    attr_accessor :host, :ws_in, :ws_out, :ws_err, :wait_thr, :ssl, :index
```